<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMeetingRoomRSV</class>
 <widget class="QDialog" name="AddMeetingRoomRSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nomeLabel">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomeLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="cognomeLabel">
      <property name="text">
       <string>Cognome</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cognomeLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="oraInizioLabel">
      <property name="text">
       <string>Ora inizio</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="oraInizioSpinBox">
      <property name="minimum">
       <number>9</number>
      </property>
      <property name="maximum">
       <number>23</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="oraFineLabel">
      <property name="text">
       <string>Ora fine</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="oraFineSpinBox">
      <property name="minimum">
       <number>9</number>
      </property>
      <property name="maximum">
       <number>23</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="indietroButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Indietro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aggiungiButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
